Extensions/3D/Model3DRuntimeObject.ts

```diff
@@ -9,9 +11,40 @@ namespace gdjs {
       rotationZ: number;
       keepAspectRatio: boolean;
       materialType: 'Basic' | 'StandardWithoutMetalness' | 'KeepOriginal';
+      originLocation:
+        | 'ModelOrigin'
+        | 'ObjectCenter'
+        | 'BottomCenterZ'
+        | 'BottomCenterY'
+        | 'TopLeft';
+      centerLocation:
+        | 'ModelOrigin'
+        | 'ObjectCenter'
+        | 'BottomCenterZ'
+        | 'BottomCenterY';
+      animations: Model3DAnimation[];
     };
   }
 
+  type FloatPoint3D = [float, float, float];
```

Comment:
This kind of stuff could be at some point moved in the game engine itself.
Same for the RuntimeObject3D.